Shared/Extensions/JellyfinAPI/TranscodeReason.swift

```diff
@@ -66,13 +66,18 @@ extension TranscodeReason: Displayable, SystemImageable {
             return L10n.videoRangeTypeNotSupported
         case .videoCodecTagNotSupported:
             return L10n.videoCodecTagNotSupported
+        case .streamCountExceedsLimit:
+            return L10n.streamCountExceedsLimit
+        @unknown default:
```

Comment:
We don't need `@unknown default` as it doesn't do anything. The SDK isn't a library in library evolution mode and extensible enums aren't in a language release yet.